### DISCUSSION TITLE
Doesn't calculate all transactions

### DISCUSSION THREAD

**Discussion by sosinovitch** [davetapley/helium-tax#31] [General]
I just used your tool for two of my hotspots, and it stopped at 200 transactions for one miner and 300 for another miner, both were short in terms of total HNT earned.

**Comment by davetapley** [comment]
Thanks for reporting, I've seen it do this before but it's hard to pin down why.

Were you able to get it to happen multiple times?
If so could you run it with [dev tools open](https://developer.chrome.com/docs/devtools/open/#console) and paste any errors or messages you see? 🙏🏻

**Reply by piot-admin** [reply]
Hi, after posting on the other thread, I checked the the version of my Chrome and it automatically downloaded the latest version. And then I tried running it again. VOILA! It worked. :) I didn't have the the opportunity to try out your proposed steps here :) But thanks for responding quick! :)

**Comment by bradranks** [comment]
I am getting lots of errors in the console:
404 - https://api.helium.io/v1/hotspots/14semXK8sLg66aRGErrGRu8PspEF6yRqASf6fHatQnkjxVfRhLK (Failed to load resource: the server responded with a status of 404 ()

Lots of CORS errors too:
Access to XMLHttpRequest at 'https://api.helium.io/v1/hotspots/1126aEKW3fF3D3AknmjtLR2xj53eCu5aUrrUtJ4MvCaTTYpzQWqg/rewards?cursor=eyJ0eG4iOiIwS0hpR1FoY0FOTjBtQnFyRExBUVpLaHIza1BkMm9ZamkxN212MmZRejhZIiwiZW5kX2Jsb2NrIjo2NTUyODksImJsb2NrIjoxMDg4MzQyLCJhbmNob3JfYmxvY2siOjEwOTkwMDB9' from origin 'https://davetapley.com' has been blocked by CORS policy: No 'Access-Control-Allow-Origin' header is present on the requested resource.

**Reply by davetapley** [reply]
Thanks for reporting @bradranks.

I believe this is caused by ongoing outage (per https://status.helium.com/), so 🤞🏻 it'll resolve itself when that does.

I've also got #26 in the pipeline to better detect/show when this happens.

**Reply by bradranks** [reply]
#AZStrong
